test/interaction.jl

```diff
@@ -4,8 +4,10 @@
         beta = 1e4
         rs = 2.0
         param = Interaction.Parameter.defaultUnit(1 / beta, rs)
-        # println(Interaction.RPA(0.01, 1, param))
-        # println(Interaction.KO(0.01, 1, param))
+        println(Interaction.RPA(1.0, 1, param))
```

Comment:
don't use println to test. 
Instead, you may compare the regularized polar to the unregularized polar at small but regular q, check if they are consistent with each other